cloud/services/networking/loadbalancers.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package networking
+
+import (
+	"net/http"
+
+	"github.com/digitalocean/godo"
+	infrav1 "sigs.k8s.io/cluster-api-provider-digitalocean/api/v1alpha2"
+)
+
+func (s *Service) GetLoadbalancer(id string) (*godo.LoadBalancer, error) {
+	if id == "" {
+		return nil, nil
+	}
+
+	lb, res, err := s.scope.LoadBalancers.Get(s.ctx, id)
+	if err != nil {
+		if res.StatusCode == http.StatusNotFound {
+			return nil, nil
+		}
+		return nil, err
+	}
+
+	return lb, nil
+}
+
+func (s *Service) CreateLoadbalancer() (*godo.LoadBalancer, error) {
+	clusterName := infrav1.DOSafeName(s.scope.Name())
+	name := clusterName + "-" + infrav1.APIServerRoleTagValue
+	request := &godo.LoadBalancerRequest{
+		Name:      name,
+		Algorithm: "round_robin",
+		Region:    s.scope.Region(),
+		ForwardingRules: []godo.ForwardingRule{
+			{
+				EntryProtocol:  "tcp",
+				EntryPort:      6443,
+				TargetProtocol: "tcp",
+				TargetPort:     6443,
+			},
+		},
+		HealthCheck: &godo.HealthCheck{
+			Protocol:               "tcp",
+			Port:                   6443,
+			CheckIntervalSeconds:   10,
+			ResponseTimeoutSeconds: 5,
+			UnhealthyThreshold:     3,
+			HealthyThreshold:       5,
+		},
```

Comment:
Should we allow the health checks options to be configured?

Reply:
addressed in this section https://github.com/kubernetes-sigs/cluster-api-provider-digitalocean/pull/140#discussion_r382952941